packages/client/src/views/SysAdmin/Config/FormConfigWizard.tsx

```diff
@@ -0,0 +1,165 @@
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { IStoreState } from '@client/store'
+import { getScope } from '@client/profile/profileSelectors'
+import { injectIntl, WrappedComponentProps as IntlShapeProps } from 'react-intl'
+import { RouteComponentProps, Redirect } from 'react-router'
+import { HOME } from '@client/navigation/routes'
+import { EventTopBar } from '@opencrvs/components/lib/interface'
+import { SettingsBlue } from '@opencrvs/components/lib/icons'
+import {
+  SecondaryButton,
+  SuccessButton
+} from '@opencrvs/components/lib/buttons'
+import { goBack } from 'connected-react-router'
+import styled from '@client/styledComponents'
+import {
+  PageNavigation,
+  TAB_BIRTH,
+  TAB_DEATH
+} from '@client/components/formConfig/PageNavigation'
+import { FormTools } from '@client/components/formConfig/formTools/FormTools'
+import { FormConfigCanvas } from '@client/components/formConfig/FormConfigCanvas'
+import { IForm, Event } from '@client/forms'
+import { getRegisterForm } from '@client/forms/register/declaration-selectors'
+import { goToFormConfigWizard } from '@client/navigation'
+import { buttonMessages } from '@client/i18n/messages'
+import { Scope } from '@client/utils/authUtils'
+
+type RouteProps = RouteComponentProps<{
+  event: string
+  section: string
+}>
+
+interface IStateProps {
+  scope: Scope | null
+  registerForm: { [key: string]: IForm }
+  event?: Event
+  section?: string | undefined
+}
+
+interface IDispatchProps {
+  goBack: typeof goBack
+  goToFormConfigWizard: typeof goToFormConfigWizard
+}
+
+const WizardContainer = styled.div`
+  margin-top: 56px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    display: none;
+  }
+`
+
+const NavigationContainer = styled.div`
+  top: 56px;
+  width: 250px;
+  position: fixed;
+  height: 100%;
+  border-right: 1px solid ${({ theme }) => theme.colors.grey300};
+  background-color: ${({ theme }) => theme.colors.white};
+`
+
+const ToolsContainer = styled.div`
+  right: 0px;
+  top: 56px;
+  width: 348px;
+  position: fixed;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 30px;
+  height: 100%;
+  border-left: 1px solid ${({ theme }) => theme.colors.grey300};
+  background-color: ${({ theme }) => theme.colors.white};
+`
+
+type IFullProps = IDispatchProps & IStateProps & IntlShapeProps & RouteProps
+
+const topBarActions = (props: IFullProps) => {
+  return [
+    <SettingsBlue key="settings" onClick={() => {}} />,
+    <SecondaryButton key="save" size="small" onClick={() => {}}>
+      {props.intl.formatMessage(buttonMessages.save)}
+    </SecondaryButton>,
+    <SuccessButton key="publish" size="small" onClick={() => {}}>
+      {props.intl.formatMessage(buttonMessages.publish)}
+    </SuccessButton>
+  ]
+}
+
+function FormConfigWizardComp(props: IFullProps) {
+  if (
+    !props.scope?.includes('natlsysadmin') ||
```

Comment:
Can we have a list of different scopes in `authUtils.ts` similar to what we have on some of the backend services like

```ts
const enum RouteScope {
  DECLARE = 'declare',
  REGISTER = 'register',
  CERTIFY = 'certify',
  PERFORMANCE = 'performance',
  SYSADMIN = 'sysadmin',
  VALIDATE = 'validate',
  NATLSYSADMIN = 'natlsysadmin'
}
```

and then maybe an exported method like

```ts
function isNationalAdministrator(scopes: Scopes): boolean
```

which you'd use here instead of the hardcoded string?

Reply:
yeah, I guess that will be good. And I think, it can be done on some of other places too.